packages/harmony/README.md

```diff
@@ -0,0 +1,87 @@
+<p align="center">
+  <img height="288px" src="./src/storybook/assets/harmonyLogoDark.png#gh-dark-mode-only">
+  <img height="288px" src="./src/storybook/assets/harmonyLogo.png#gh-light-mode-only">
+
+  <p align="center">
+    Harmony is design system focused on collaboration, reusability, and scalability.
+  </p>
+  <p align="center">
+    It aims to harmonize code and Figma, provide a shared language for designers and developers, and provide consistent, reusable components for use across our platforms.
+  </p>
+  <p align="center">
+    built with ❤️ from the team <a href="https://audius.org">@Audius</a>.
+  </p>
+</p>
+
+<br />
+<br />
+
+## Docs
+
+Full documentation can be found here: [Harmony Docs](https://harmony.audius.co)
+
+## Installation
+
+Install `@audius/harmony` required peer dependencies:
+
+```bash
+npm install --save @emotion/react @emotion/styled @radix-ui/react-slot @react-spring/web classnames lodash react-lottie react-merge-refs react-perfect-scrollbar react-use react-use-measure
+```
+
+Then install `@audius/harmony`
+
+```bash
+npm install --save @audius/harmony
+```
+
+## Setup
+
+Import styles exported by Harmony
+
+```ts
```

Comment:
do we want to make a callout that this is optional? like if you are using sass/css, use this? i think we havent fully migrated yet so maybe we keep it, but potentially a callout we just get ahead on?

Reply:
yea for the time being we need this, but in the future if we have everything using emotion (and drop css variables support?) we can remove